Readme.md

```diff
@@ -0,0 +1 @@
+#Method mockedFetched is a custom method 
```

Comment:
Hey please add more elaborate Readme. Add more sections about, 
how to install
how to use
What advantage this library has
How to contribute